docker-compose.linux-dev.yaml

```diff
@@ -1,4 +1,5 @@
 version: "3.2"
-  services:
-    localstack:
-      extra_hosts: - "host.docker.internal:host-gateway"
\ No newline at end of file
+services:
+  localstack:
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
Have we tried if adding this line to a non linux `docker-compose.yaml` file breaks it?

Reply:
I will test it with the features I'm doing right now. Merging meanwhile so that linux users have a working version